src/calendar/index.js

```diff
@@ -65,6 +65,8 @@ class Calendar extends Component {
     enableSwipeMonths: PropTypes.bool,
     /** Disable days by default. Default = false */
     disabledByDefault: PropTypes.bool,
+    /** Disable all touch events for disabled days. Default = false */
+    disableAllTouchEventsForDisabledDays: PropTypes.bool,
```

Comment:
`disableAllTouchEventsForDisabledDays` is a BasicDay component prop. There is no need to declare it as the Calendar get all Day props (see line #31).